### DISCUSSION TITLE
What are Secondary Faces?

### DISCUSSION THREAD

**Discussion by BrianBinFL** [joeycastillo/Sensor-Watch#280] [Q&A] [answered]
I received my beautiful red boards today. I immediately installed one into one of the several variants of F91-W that I had pre-ordered waiting for today to arrive. I used the Sensor Watch Builder to build a firmware with the faces that I wanted, and installed the faces onto the board no problem. I put the watch back together, and everything works great.

But in the process I was reminded that in the Sensor Watch Builder there is a dialog for "Secondary Faces". You can click on a face that you've added to "Faces" and move it to "Secondary Faces". I've read and I've searched but I don't find anything that explains what a Secondary Face is. 

The documentation is very well written, and very comprehensive, so I'm sure I've just overlooked it. Please accept my apologies for that failure, but a pointer to the answer would be most appreciated.

Thanks!

**Comment by BrianBinFL** [comment, marked as answer]
So I solved my own question by playing around with the emulator. I'll go ahead and answer it here for the benefit of anybody else that comes along with the same question.

You cycle through the regular faces on the watch by pressing Mode. To access the Secondary Faces you long-press Mode and then press Mode to cycle through the Secondary faces.

The obvious use for this is to put the faces that can change data (time setting, fine tune, nanosec, etc.) as Secondary Faces so you only see/access them when you want to change something and long-press the Mode button.

**Reply by wryun** [reply]
Yes - also, if you cycle off the 'end' of the secondary faces, you go back to the start of the faces (not the start of the secondary faces).

This was something that got added after the initial run of boards as a compile time option, and I think Joey decided at the last minute to put it as the default: https://github.com/joeycastillo/Sensor-Watch/commit/d323b7991622202d3258ee2dc1359034df7bbce8

The other thing that might not have been clear is that a long press of mode on any face will usually bring you back to the first face (unless it's overridden).

**Reply by BrianBinFL** [reply]
Thanks for the interesting tips Wryun!
